examples/Arduino_Logger/Arduino_Logger.ino

```diff
@@ -2,17 +2,16 @@
  * Define Constants
  ****************************************/
 namespace {
+  bool flag = true;
   const char * USER_AGENT = "UbidotsESP8266"; // Assgin the user agent
-  const char * VERSION =  "1.0"; // Assign the version 
+  const char * VERSION =  "1.0"; // Assign the version
   const char * METHOD = "POST"; // Set the method
-  const char * TOKEN = "Put_your_Ubidots_token_here"; // Assign your Ubidots TOKEN
-  const char * DEVICE_NAME = "telemetry"; // Assign the desire device name 
-  const char * DEVICE_LABEL = "ESP8266"; // Assign the device label 
-  const char * VARIABLE_LABEL_1 = "temperature"; // Assign the variable label 
-  const char * VARIABLE_LABEL_2 = "humidity"; // Assign the variable label
+  const char * TOKEN = "........"; // Assign your Ubidots TOKEN
+  const char * DEVICE_LABEL = "ESP8266"; // Assign the device label
+  const char * VARIABLE_LABEL = "temp"; // Assign the variable label
 }
 
-char command[700]; // command
+char* command = (char *) malloc(sizeof(char) * 128);
```

Comment:
This should be on line 33, once you call the free() method you lose this memory reservation